cli.sh

```diff
@@ -1147,7 +1147,8 @@ update_shared_hooks_in() {
 
     # split on comma and newline
     IFS=",
-    "
+"
```

Comment:
This is super crucial, as it splits also for whitespaces...

Reply:
Nice one, good catch!